talking-about-their-pronouns.md

```diff
@@ -0,0 +1,38 @@
+# Asking someone's pronouns
+
+## Over Slack / in conversation
+
+What: Sorry, just to check so I can get them right, do you mind me asking what your pronouns are?
+
+# Correcting someone who misgenders someone else
+
```

Comment:
```suggestion
:warning: If the person is in the conversation, they may be happy to correct it themselves, or leave it be and handle it via DMs, so exercise caution in doing this if they are present. They are, after all, the authority on how they wish to be addressed!
```

Reply:
Yep, I always go and check Slack or something unless we've explicitly had that conversation before